root/static/scripts/common/entity.js

```diff
@@ -177,32 +197,16 @@ const formatTrackLength = require('./utility/formatTrackLength');
         }
     }
 
-    CoreEntity.prototype.template = _.template(
-        "<% if (data.editsPending) { %><span class=\"mp\"><% } %>" +
-        "<% if (data.nameVariation) { %><span class=\"name-variation\" title=\"<%- data.name %>\"><% } %>" +
-        "<a href=\"/<%= data.entityType %>/<%- data.gid %>\"" +
-        "<% if (data.target) { %> target=\"_blank\"<% } %>" +
-        "<% if (data.sort_name) { %> title=\"<%- data.sort_name %>\"" +
-        "<% } %>><bdi><%- data.creditedAs || data.name %></bdi></a>" +
-        "<% if (data.comment) { %> " +
-        "<span class=\"comment\">(<%- data.comment %>)</span><% } %>" +
-        "<% if (data.video) { %> <span class=\"comment\">" +
-        "(<%- data.videoString %>)</span><% } %>" +
-        "<% if (data.nameVariation) { %></span><% } %>" +
-        "<% if (data.editsPending) { %></span><% } %>",
-        {variable: "data"}
-    );
-
-    class Editor extends CoreEntity {}
+    class Editor extends CoreEntity {
+        html() {
```

Comment:
I'm not positive, but I don't think this `html` method is ever called anywhere. I believe this was just a stub class to allow the editor search autocompletes to work (and that doesn't render editor links either). I've ported it over just in case.